autosubmitconfigparser/config/configcommon.py

```diff
@@ -1185,13 +1210,73 @@ def check_conf_files(self, running_time=False, force_load=True, no_log=False):
+
+        :return: Error message if any job exceeds the platform's wallclock time, otherwise an empty string.
+        :rtype: str
+        """
+
+        def _calculate_wallclock(wallclock: str) -> int:
+            hours, minutes = map(int, wallclock.split(":"))
+            return timedelta(hours=hours, minutes=minutes).total_seconds()
+
+        default_wallclock = _calculate_wallclock(self.experiment_data.get("CONFIG", {}).get("JOB_WALLCLOCK", "24:00"))
+        err_msg = ""
+        jobs = self.experiment_data.get("JOBS", {})
+        platforms = self.experiment_data.get("PLATFORMS", {})
+        wallclock_per_platform = {}
+
+        for platform_name in platforms.keys():
+            wallclock_per_platform[platform_name] = _calculate_wallclock(platforms[platform_name].get("MAX_WALLCLOCK",
+                                                                                                      self.experiment_data.get(
+                                                                                                          "CONFIG",
+                                                                                                          {}).get(
+                                                                                                          "JOB_WALLCLOCK",
+                                                                                                          "24:00")))
+
+        for job_name, job_data in jobs.items():
+            platform_wallclock = wallclock_per_platform.get(job_data.get("PLATFORM", ""), default_wallclock)
+            total_seconds = _calculate_wallclock(job_data.get("WALLCLOCK", "00:01"))
+            if total_seconds > platform_wallclock:
+                err_msg += f"Job {job_name} has a wallclock {total_seconds}s time greater than the platform's {total_seconds}s wallclock time\n"
+        return err_msg
+
+    def validate_jobs_conf(self) -> str:
+        """
+        Validate the job configurations.
+
+        :return: Error message if any validation fails, otherwise an empty string.
+        :rtype: str
+        """
+        err_msg = self.validate_wallclock()
+        return err_msg
+
+    def validate_config(self, running_time: bool) -> bool:
+        """
+        Check if the configuration is valid.
+
+        :param running_time: Indicates if the validation is being performed during runtime.
+        :type running_time: bool
+        :raises AutosubmitCritical: If any validation error occurs during runtime.
+        """
+        error_msg = self.validate_jobs_conf()
+        if error_msg and running_time:
+            raise AutosubmitCritical(error_msg, 7014)
+        elif error_msg and not running_time:
+            Log.printlog(f"Fix your invalid configuration before running!:{error_msg}", 7014)
+            return False
+        else:
+            Log.result('Validation OK (WIP, not all configuration is validated)')
+            return True
```

Comment:
Maybe the if/else could be decomposed, like,

```py
        error_msg = self.validate_jobs_conf()
        if not error_msg:
            Log.result('Validation OK (WIP, not all configuration is validated)')
            return True

        if running_time:
            raise AutosubmitCritical(error_msg, 7014)

        Log.printlog(f"Fix your invalid configuration before running!:{error_msg}", 7014)
        return False
```

but not really important, they produce the something...

Reply:
But maybe the messages could be improved a bit?

"Validation OK (WIP, not all configuration is validated)" → "Partial configuration validated correctly"
"Fix your invalid configuration before running!:{error_msg}"" → "Invalid configuration. You must fix it before running your experiment: {...}"